test/gtest/common/mem_buffer.cc

```diff
@@ -153,31 +153,48 @@ size_t mem_buffer::get_bar1_free_size()
 {
     /* All gtest CUDA tests explicitly assume that all memory allocations are
      * done on the device 0. The same assumption is followed here. */
-    size_t availible_size = SIZE_MAX;
+    size_t available_size = SIZE_MAX;
```

Comment:
why removed static?

Reply:
We discussed it on the call - depending what run on the machine, bar1 size can change. It is safer to query bar1 size on each allocation.

Reply:
Yes, it is can and depends on machine where it runs. CUDA tl already calls init/cleanup and this call increases reference counter. Only if you run as a stand alone benchmark it may cause an actual init/cleanup calls.

Reply:
ok